client/certutils.go

```diff
@@ -0,0 +1,12 @@
+// +build !windows
+
+package midgardclient
+
+import (
+	"crypto/x509"
+)
+
+// GetSystemCertPool just calls the Go implementation
+func GetSystemCertPool() (*x509.CertPool, error) {
```

Comment:
This should not be public. If you plan to reuse it somewhere else, then this function should be in tglib

Reply:
We were using it from enforcerd. I will move it to tglib.